COPYRIGHT.txt

```diff
@@ -110,6 +110,11 @@ Copyright: 2007, Starbreeze Studios
  2014-2019, Godot Engine contributors.
 License: Expat and Zlib
 
+Files: ./thirdpart/assimp/
+Comment: Open Asset Import (Assimp)
```

Comment:
In their LICENSE it's `Open Asset Import Library (assimp)`, so we should probably use the exact same string.

Reply:
In other places they do capitalize the A in Assimp though, so I don't know. But add "Library" at least.

Reply:
Update.